packages/perseus/src/styles/widgets/numeric.less

```diff
@@ -0,0 +1,19 @@
+.input-with-examples-tooltip {
```

Comment:
While these styles used to live in perseus-renderer.less, I think it makes more sense in a numeric file now. We're going to be deleting Input Number, and Expression no longer shows answer formats. 

Unfortunately, the tooltip has a reset, so we need to redefine a couple of the text styles here.